vnext/Directory.Build.targets

```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
 
+  <Import Condition="Exists($([MSBuild]::GetPathOfFileAbove('Directory.Build.targets', '$(MSBuildThisFileDirectory)../')))" Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.targets', '$(MSBuildThisFileDirectory)../'))" />
```

Comment:
This will no-op for the published package, but that seems like what we want? Might be worth leaving a comment about that though.

Reply:
Added the comment and did the same for the .props file we have...